legal-api/src/legal_api/services/document_meta.py

```diff
@@ -0,0 +1,154 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This provides the service for filings documents meta data."""
+
+from enum import Enum
+
+from legal_api.models import Filing
+from legal_api.utils.legislation_datetime import LegislationDatetime
+
+
+class DocumentMetaService():
+    """Provides service for document meta data."""
+
+    class DocumentType(Enum):
+        """Define an enum of document types."""
+
+        REPORT = 'REPORT'
```

Comment:
We now have 3 words for the same thing (output, document, report). Is it possible to choose just one?

Reply:
We can, the ticket mentioned to exclude receipts as they were from a different api so I just followed that.

Reply:
@severinbeauvais so I wanted to use documents as I wanted this document service to potentially handle more than outputs/reports as needed and it populates the documents array in the filing json.

The word report is what is also mainly used in the API, so I am not sure what is best to refactor here.

@lekshmimallika-aot any thoughts?

Reply:
Looks fine to me